client-tools/Dump/arangodump.cpp

```diff
@@ -80,6 +81,10 @@ int main(int argc, char* argv[]) {
         []<typename T>(auto& server, TypeTag<T>) {
           return std::make_unique<T>(server);
         },
+        [](ArangoDumpServer& server, TypeTag<BumpFileDescriptorsFeature>) {
+          return std::make_unique<BumpFileDescriptorsFeature>(
+              server, "--descriptors-minimum");
```

Comment:
What is the point to have ability to specify different name for same option for different binaries?